bezier-rs/lib/src/lib.rs

```diff
@@ -721,6 +721,48 @@ impl Bezier {
 		});
 		result
 	}
+
+	/// Returns list of `t`-values representing the inflection points of the curve.
+	/// The inflection points are defined to be points at which the second derivative of the curve is equal to zero.
+	/// The formulas are from  [the inflection section](https://pomax.github.io/bezierinfo/#inflections) of Pomax's bezier curve primer.
+	pub fn unrestricted_inflections(&self) -> Vec<f64> {
+		match self.handles {
+			// There exists no inflection points for linear and quadratic beziers.
+			BezierHandles::Linear => Vec::new(),
+			BezierHandles::Quadratic { .. } => Vec::new(),
+			BezierHandles::Cubic { .. } => {
+				// Axis align the curve.
+				let translated_bezier = self.translate(-self.start);
+				let angle = translated_bezier.end.angle_between(DVec2::new(1., 0.));
+				let rotated_bezier = translated_bezier.rotate(angle);
+				match rotated_bezier.handles {
```

Comment:
Use `if let` instead of `match`.

Reply:
@linda-zheng thanks for the changes. For future reference, please click "Resolve conversation" for anything that's been taken care of so it simplifies the view of what's left to be done. Thanks for the 👍 though, I've gone ahead and clicked resolve on all the ones you gave that reaction for.